pkgs/tools/security/nwaku/default.nix

```diff
@@ -0,0 +1,97 @@
+{
+  stdenv, lib, fetchurl, fetchFromGitHub,
+  nim, cmake, which, writeScriptBin,
+  # Options: wakunode1, wakunode2, wakubridge
+  makeTargets ? [ "wakunode2" ],
+  # WARNING: CPU optmizations that make binary not portable.
+  nativeBuild ? false
+}:
+
+assert nim.version == "1.6.12";
+
+stdenv.mkDerivation rec {
+  pname = "nwaku";
+  version = "0.16.0";
+  commit = "03f0de36";
+  name = "${pname}-${version}-${commit}";
```

Comment:
```suggestion
```

Reply:
No, we include the commit in the version of the node.

Reply:
We don't add that to name.

Reply:
I can remove it from `name`, but I will keep `commit` anyway for the sake of fake git for `git rev-parse`.

Reply:
The contributing guide should mention how pname and version and name should be handled and we normally don't write commit hashes into name/version if we are using a release tag.

Reply:
That's fine. I can remove it, but I will still keep it for fake `git rev-parse`, as I said.